rubric_data/views.py

```diff
@@ -18,6 +19,7 @@ def course_data(request, course_id):
     
     access_token = get_oauth_token(request)
     request_context = RequestContext(**settings.CANVAS_SDK_SETTINGS, auth_token=access_token)
+    cache_user_id_course_id = f"{request.user}{course_id}"
```

Comment:
Note that the string interpolation of the `request.user` object returns the `username` by default (see [implementation](https://github.com/django/django/blob/02c59b7a4355fda8c99224b5de9c0a3929bffe22/django/contrib/auth/base_user.py#L63)). I had to look this up because I wasn't sure if it would return that or the ID by default. Just FYI.